### DISCUSSION TITLE
Docker Compose & Traefik Proxy

### DISCUSSION THREAD

**Discussion by DannyKuk** [custompbx/custompbx#9] [General]
Good Day,
I am having on my server the Traefik proxy to handle all incoming requests. 
Traefik also provides SSL certificates to all of my services, that's where the issue starts.
When routing Traefik to the server, it uses https, the port 8080 and the prefix /cweb.
Now when trying to connect I run into an internal server error: http: TLS handshake error from 172.18.0.4:42726: remote error: tls: bad certificate

Is there a way I leave the certificate check off the docker container? 
The connection between Traefik as well as custompbx is only internally so the only certificate needed (on the domain) will be provided by Traefik.

Thanks in advance!

**Comment by zusrut** [comment]
hi. your post looks like ads for the proxy service =) 
TLS is enabled by default and if it will not find cert and key it will generate them. So now you can set `webCert` to empty string in func `startServers` in the file `src/custompbx/main.go`  and rebuild docker image.

I will rework web server tls settings later.

**Comment by zusrut** [comment]
pushed changes - in `config.json` set `secure` option for webserver to `false`

**Comment by DannyKuk** [comment]
Thank you very much, my post was not intended to be advertisement. I am sorry if it seems like.

**Comment by DannyKuk** [comment]
I just tried the new version with the config. I am running in new errors when the webserver secure is set to false.

Docker logs:
2024/06/08 09:06:59 Asset cweb/styles.6697ff13009d4d3e.css not found
2024/06/08 09:06:59 Asset cweb/runtime.0159572f4929421f.js not found
2024/06/08 09:06:59 Asset cweb/polyfills.11478848d8816ece.js not found
2024/06/08 09:06:59 Asset cweb/main.265e157083ae8f40.js not found 

The web server is showing only a white screen and the browser console shows:
[Error] SyntaxError: Unexpected token '<'
	(anonyme Funktion) (runtime.0159572f4929421f.js:1)
[Error] SyntaxError: Unexpected token '<'
	(anonyme Funktion) (polyfills.11478848d8816ece.js:1)
[Error] SyntaxError: Unexpected token '<'
	(anonyme Funktion) (main.265e157083ae8f40.js:1)
	

Am I missing something out?
Thanks in advance, appreciate all the effort!

**Reply by zusrut** [reply]
my bad - now fixed

**Comment by zusrut** [comment]
try with last commits

**Comment by DannyKuk** [comment]
First of all excuse the late answer.

Secondly I am really thankful for the pbx and all the support, it works like a dream now! 
I really appreciate everything and am happy to use the pbx! 🥇
